### ISSUE TITLE
Embedded subtitles used instead of external

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#2052]
User jkwu mentioned a regression in v8+ at https://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=14007&start=10#p41144

@valib

**Comment by SubJunk** [comment]
@valib any interest in looking into this?

**Comment by valib** [comment]
@SubJunk I will check it.

**Comment by SubJunk** [comment]
I looked into it yesterday and it seems like it might be a case where we fixed a bug, but the bug had been causing a good user experience. It seems like that renderer for the user is set up to receive external subtitles, but then when we try to construct the headers, we find that the renderer config has no subtitle header defined.

I guess the old version that worked for the user (7.9.0) had been transcoding the external subtitles.

Those are my rough notes on it anyway. I'm interested to know your findings too.
